.github/workflows/continuous-deployment.yml

```diff
@@ -294,7 +301,9 @@ jobs:
             |---------------------------------|------------------------|
             |<span aria-hidden="true">🔨</span> Latest commit | [${{ matrix.deployment.sha }}](https://github.com/${{ github.repository }}/commit/${{ matrix.deployment.sha }}) |
             |<span aria-hidden="true">🔍</span> Latest deploy log | [${{ steps.job-url.outputs.html_url }}](${{ steps.job-url.outputs.html_url }}) |
+            |<span aria-hidden="true">🕒</span> Last deployed at | ${{ steps.current-time.outputs.readableTime }} |
             |<span aria-hidden="true">😎</span> **Deployment** | [https://${{ matrix.deployment.name }}.ecamp3.ch/](https://${{ matrix.deployment.name }}.ecamp3.ch/) |
```

Comment:
If you add more info, then I think this should be the first line :)

Reply:
Sure. Which order would you prefer? How about:


|  Name | Link |
|---------------------------------|------------------------|
|<span aria-hidden="true">😎</span> **Deployment** | [https://pr3348.ecamp3.ch/](https://pr3348.ecamp3.ch/) |
|<span aria-hidden="true">🔑</span> Login | `test@example.com` / `test` |
|<span aria-hidden="true">🕒</span> Last deployed at | 2020-01-01T00:30:15.000Z |
|<span aria-hidden="true">🔨</span> Latest commit | [bdcc16942346c5de69b202a3b7d2e950aebcc145](https://github.com/ecamp/ecamp3/commit/bdcc16942346c5de69b202a3b7d2e950aebcc145) |
|<span aria-hidden="true">🔍</span> Latest deploy log | [https://github.com/ecamp/ecamp3/actions/runs/4414807795/jobs/7736990193](https://github.com/ecamp/ecamp3/actions/runs/4414807795/jobs/7736990193) |
|<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />![QR Code](https://api.qrserver.com/v1/create-qr-code/?size=180x180&ecc=H&format=svg&qzone=4&color=5-72-97&data=https://pr3348.ecamp3.ch/)<br /><br />_Use your smartphone camera to open QR code link._</details> |